lib/src/input_fields/primary_input_field.dart

```diff
@@ -17,6 +17,10 @@ class MUIPrimaryInputField extends StatefulWidget {
     this.textStyle = const TextStyle(),
     this.hintStyle = const TextStyle(fontWeight: FontWeight.w400),
     this.isObscure = false,
+    this.suffixIcon,
+    this.prefixIcon,
+    this.suffixIconAction,
+    this.prefexIconAction,
```

Comment:
Would you please rename it as `suffixIconOnPressed` & `prefixIconOnPressed` because we maintain that practice in some other widgets.

Reply:
Sure